### DISCUSSION TITLE
How to update @rsdoctor/rspack-plugin to fit rspack@1.0.0-alpha.5

### DISCUSSION THREAD

**Discussion by Zzzdy-code** [web-infra-dev/rsdoctor#428] [Q&A]
![image](https://github.com/user-attachments/assets/738e54ff-7590-4203-b4a7-fb1d051f02ee)

**Comment by Zzzdy-code** [comment]
![image](https://github.com/user-attachments/assets/5c9de772-18bd-4173-bf78-b3594c32d759)

**Comment by yifancong** [comment]
Got this problem, I'll fix this.

**Comment by yifancong** [comment]
@Zzzdy-code Since I cannot reproduce this problem, need you try version `v0.3.10-beta.0` to see if it resolves the issue.

**Comment by Zzzdy-code** [comment]
@easy1090 It doesn't seem to solve the problem
![image](https://github.com/user-attachments/assets/2477253d-d89b-468d-994a-7d501d40683c)

**Reply by Zzzdy-code** [reply]
It can be reproduce by run
```
npm add -D @rspack/core@^1.0.0-alpha.5
npm add -D @rspack/cli@^1.0.0-alpha.5
npm add -D @rspack/dev-server@^1.0.0-alpha.5
npm add -D @rspack/plugin-react-refresh@^1.0.0-alpha.5
npm add -D @rsdoctor/rspack-plugin@^0.3.10-beta.0  //It will report error
```

**Reply by yifancong** [reply]
ok, I'll try this.

**Reply by yifancong** [reply]
Use npm install --legacy-peer-deps° or pnpm to solve it first. I'll publish fixed version this week.

**Reply by yifancong** [reply]
@Zzzdy-code please try the 0.3.11 version ❤

**Reply by Zzzdy-code** [reply]
Successfully solved the problem.Thanks!❤
